spec.md

```diff
@@ -289,17 +289,21 @@ The error code `OPERATION_PENDING_FOR_VOLUME` may be returned by the plugin in t
 
 #### `GetSupportedVersions`
 
-A Plugin SHALL reply with a list of supported CSI versions. The initial version of the CSI specification is 0.1.0 (in *major.minor.patch* format).
+A Plugin SHALL reply with a list of supported CSI versions.
+The initial version of the CSI specification is 0.1.0 (in *major.minor.patch* format).
 A CO MAY execute plugin RPCs in the manner prescribed by any such supported CSI version.
 The versions returned by this call are orthogonal to any vendor-specific version metadata (see `vendor_version` in `GetPluginInfoResponse`).
 
+NOTE: Changes to this RPC should be approached very conservatively since the request/response protobufs here are critical for proper client-server version negotiation.
```

Comment:
Thanks for the feedback. I'm not crazy about immutable protos. We purposefully didn't declare fields using `required` syntax, and instead included `REQUIRED` and `OPTIONAL` in the docs for each field. I'm also not crazy about the `XyzExRequest` nomenclature.